docs/specs/values.md

```diff
@@ -25,6 +25,13 @@ for example:
 Note sometimes you might find color values with 4 components (the 4th being alpha)
 but most players ignore the last component.
 
+<h2 id="hexcolor">Hex Color</h2>
+Colors represented as a "#"-prefixed string, with one or two hexadecimal digits per
+RGB component.
+
+* {lottie_hexcolor: #F00}
```

Comment:
Huh I tried on lottie-web and it seemingly supports any valid CSS color (I tried `#rgb`, `#rrggbb`, `#rgba`, `#rrggbbaa`, `rgb()`, `rgba()`, and color names). How much of that do we want to support?

Reply:
Good point, I'm guessing lottie-web passes everything through to CSS so it supports all CSS values.

Since Bodymovin only exports in #rrggbb form, let's stick to that for now.

Updated.